COMPONENT_GUIDE.md

```diff
@@ -252,3 +252,39 @@ Text
 
 </Caution>
 ```
+
+## Related Resources
+
+The related resources component is controlled by specific Frontmatter slugs that
+are defined on a page.
+
+By setting the Frontmatter for `resources` and `tags` you can control what is populated
+in this component. Please review the [Style Guide](STYLE_GUIDE) for further details
+on these slugs.
+
+### Maximum references
+
+This component allows a maximum of 5 resources.
+
+### Resource population logic
+
+If no resources or tags are available in the page Frontmatter the component will fallback
+to using the page title as the search query term but you will
+always have 5 results, assuming Swiftype can return 5 results for that query.
+
+### Order of priority
+
+The order of priority for populating content in this component is driven by:
+
+1. Any resources defined in the page Frontmatter.
+2. Any tags defined in the the page Frontmatter will return send a search query term and return results from Swifttype.
+3. Any Swifttype results that match the page title.
```

Comment:
This is sorta a 2b if you want to be totally accurate. We don't send the page title as a query unless there are no tags for the current page and we really only send a single query to Swiftype per page. Its pretty much just an if/else condition.